pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: io
 description: >
   Utilities for the Dart VM Runtime.
-version: 0.2.1
+version: 0.3.0-dev
```

Comment:
Unless I mis-understand, this can simply be `0.2.2`, right?

Adding a `shellSplit` doesn't break existing `io: ^0.2.0` users?

Reply:
The changelog lists existing changes under a 0.3.0 heading, I'm just updating the pubspec version to match that.

Reply:
Oh, I missed that. LGTM!